bedrock/newsletter/management/commands/update_newsletter_data.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import print_function
+
+from django.core.management.base import BaseCommand, CommandError
+
+import basket
+
+from bedrock.newsletter.models import Newsletter
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument('-q', '--quiet', action='store_true', dest='quiet', default=False,
+                            help='If no error occurs, swallow all output.'),
+
+    def handle(self, *args, **options):
+        newsletters = basket.get_newsletters()
+        if not newsletters:
+            raise CommandError('No data from basket')
+
+        Newsletter.objects.all().delete()
```

Comment:
I was concerned about race conditions here at first, but then I remembered that this is being run in a separate container that isn't serving traffic, and that the web containers will update the entire DB atomically, so this shouldn't be an issue.